### PR TITLE
setup: Publish a Github release using task-boot

### PR DESCRIPTION
**Pull request by La0**
Depends on https://github.com/mozilla/community-tc-config/pull/180

### CODE REVIEW

**Comment by La0** [comment]
I added the moz-tools-bot account & setup the secret, we only need the role setup from above PR

**Comment by La0** [comment]
It works: [test-bastien release published](https://github.com/mozilla/code-review/releases/tag/test-bastien)

File: .taskcluster.yml [hunk 1]
```diff
@@ -527,3 +527,39 @@ tasks:
...
           owner: bastien@mozilla.com
           source: https://github.com/mozilla/code-review
+
+    - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+      then:
+        $let:
+          version: {$eval: 'head_branch[10:]'}
+        in:
+          taskId: {$eval: as_slugid("release")}
+          dependencies:
```
Comment (marco-c) [inline .taskcluster.yml]: Maybe we should depend on the test tasks too, so we only release when everything is green

File: .taskcluster.yml [hunk 2]
```diff
@@ -527,3 +527,39 @@ tasks:
...
+            - {$eval: as_slugid("backend_build")}
+            - {$eval: as_slugid("bot_build")}
+            - {$eval: as_slugid("events_build")}
+            - {$eval: as_slugid("frontend_build")}
+            - {$eval: as_slugid("integration_build")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hour'}
```
Comment (marco-c) [inline .taskcluster.yml]:
```suggestion
          deadline: {$fromNow: '2 hours'}
```

File: .taskcluster.yml (hunk 1 above)
Comment (La0) [reply .taskcluster.yml]: The build steps already depend on the tests, so we should be fine

**Pull request merged**